src/hooks/invoices.hooks.ts

```diff
@@ -0,0 +1,31 @@
+import { useMutation, useQuery } from 'react-query';
+import InvoicesService from '../services/invoices/invoicesService.ts';
+import { CreateInvoiceDto, UpdateInvoiceDto } from '../services/invoices/types';
+
+const INVOICE_KEY = 'invoice';
+
+export const useGetInvoices = async () => {
+  useQuery({
+    queryKey: [INVOICE_KEY],
+    queryFn: async () => InvoicesService.fetchAllInvoices(),
```

Comment:
No need for async here (apliccable to all querFns here), as InvoicesService returns Promise anyway